extension/docs/WebComponentsIntegrationGuide.md

```diff
@@ -144,3 +144,179 @@ Pass the `origin key` to your front end. You might use this origin key to be abl
 
 > Note: First 2 steps are optional if origin key and payment methods has been already cached by the merchant server.
  
+## Step 3: Make a payment
+After the shopper submits their payment details or chooses to pay with a payment method that requires a redirection,
+the Adyen Web Components will generate a `makePaymentRequest`. Consult [Adyen documentation](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/payments) to see which parameters 
+are necessary for the current payment request.
+
+[Update CTP payment](https://docs.commercetools.com/http-api-projects-payments#update-payment) with `makePaymentRequest` custom field. 
+```json
+{
+  "version": "PAYMENT_VERSION",
```

Comment:
Can we use the examples from Adyen documentation? It's better than this one. https://docs.adyen.com/checkout/components-web#step-3-make-a-payment Also it's easier to follow if they are the same. Also please don't add all data so it will be easier to read, here it's not readable, to make it readable you need to use example request JSON before showing the CTP representation. 🙏

Reply:
See comment above: https://github.com/commercetools/commercetools-adyen-integration/pull/335#discussion_r442759463. I was never be able to make payment with the example, so that's why I didn't want to put sth which does not reflect reality.